report/2.Analysis/Requirements.tex

```diff
@@ -0,0 +1,39 @@
+\section{Requirements}
+In order to be able to conclude upon the success of the project, a fixed set of requirements must be formulated.
+These requirements will serve as a guideline in the conclusion of the report and as markers throughout the project. 
+To formally specify the requirements, the MoSCoW analysis model will be utilized. 
+The MoSCoW model specifies which requirements \textbf{M}ust, \textbf{S}hould, \textbf{C}ould, and \textbf{W}on't be fulfilled to deem the project successful.
+
+\textbf{Must have}
+\begin{itemize}
+\item ability to recognize a moving target in front of the device using an attached camera
+\item ability to hit a target moving with a constant speed along a horizontal axis
```

Comment:
enten "track" eller specificering af hvad den skal "hittes" med.

Reply:
evt specificer noget så specifikt som "be able to follow a ball, with a laser, dropped from the ceiling, 3 meters from the device" (eller måske 4 meter? jeg tænker sådan cirka længden af vores rum. men jo længere væk, jo lettere er det jo.) 

Vi kan evt lave en liste af "hard requirements" som er helt binære "ja nej" og bliver fremhævet og argumenteret for hvorfor disse tal giver mening og hvorfor det er noget vi vil teste ud fra. Nærmest test-cases, så vi har "vi vil gerne kunne gøre præcis dette scenarie og så skal den gøre dette. Det virker reasonable pga dette." og så nder dem disse requirements som du har nu, som jo uanset hvad er mere vague.

Reply:
Jeg tænker at det er en super god idé at lave en liste med ja-nej elementer til listen. Synes at vi skal brainstorme nogle ting som gruppe til det, så alle er på samme side.